compiler/resolution/virtualDispatch.cpp

```diff
@@ -966,7 +966,8 @@ static void checkMethodsOverride() {
             }
           }
 
-          if (okKnown && ok == false) {
+          if (okKnown && ok == false &&
+              (developer == true || fn->isUserDefined())) {
```

Comment:
This turns on override checking more generally, for developers, right? Are we ready for that? Should we be concerned developers will be surprised by override errors during development?

Reply:
I think I understand now - it's filtering out override checking for compiler-generated functions unless --developer. Doesn't turn it on more generally.

Reply:
(or maybe in the conditional calling checkOverrides)